### ISSUE TITLE
[FEATURE]Could you add a feature for dynamically binding tag placeholders?

### ISSUE THREAD

**Issue by luy710** [frangoteam/FUXA#1109]
This functionality would enable greater flexibility in reusing layouts where, for instance, within a page's tags we have references like A.{device}.value1, A.{device}.value2, and A.{device}.value3. By passing or switching the value of the device variable, we could achieve different configurations for various devices.

**Comment by unocelli** [comment]
Hi, I agree, fuxa should better integrate the concept with placeholders

**Comment by Jue07** [comment]
Hi, is this feature already available? if so, on what version is this placeholder available? what I found after installing Fuxa on several computers not all have this placeholder even though I have used the latest version of Fuxa which is v 1.1.18.

![WhatsApp Image 2024-05-02 at 09 11 28](https://github.com/frangoteam/FUXA/assets/107406141/bd0c4a18-f1d6-4b4a-bd9f-120201514f5b)

![WhatsApp Image 2024-05-02 at 09 11 54](https://github.com/frangoteam/FUXA/assets/107406141/c8338933-0fbe-429e-ab8b-219709f95e17)
